tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -555,7 +564,25 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
     includeCommitCharacter = true,
     compat = Map(
       "2" -> "",
-      "3.3" -> """|using (commit: '')
+      "3.3.0" -> """|using (commit: '')
+                    |unsafeExceptions scala (commit: '')
+                    |unchecked scala (commit: '')
+                    |unsafe - scala.caps (commit: '')
+                    |unsafeNulls - scala.runtime.stdLibPatches.language (commit: '')
+                    |""".stripMargin,
+      "3.3.1" -> """|using (commit: '')
+                    |unsafeExceptions scala (commit: '')
+                    |unchecked scala (commit: '')
+                    |unsafe - scala.caps (commit: '')
+                    |unsafeNulls - scala.runtime.stdLibPatches.language (commit: '')
+                    |""".stripMargin,
+      "3.3.2" -> """|using (commit: '')
+                    |unsafeExceptions scala (commit: '')
+                    |unchecked scala (commit: '')
+                    |unsafe - caps (commit: '')
```

Comment:
```suggestion
                    |unsafe - caps (commit: '')
```
Scala is not printed here, there is a specific logic in the printer that is hard to make work and is working fine with the presentation compiler module itselg.